src/components/GameAbout/GameAbout.module.scss

```diff
@@ -1,32 +1,119 @@
+.container {
+  display: flex;
+  width: 75rem;
+  padding: var(--space-2xl) 7.5rem;
```

Comment:
```suggestion
```

I think adding padding or margin here is too opinionated. We should let `classnames?.container` passed into `GameAbout` be used for these styles